xml_converter/src/xml_converter.cpp

```diff
@@ -217,80 +249,72 @@ void process_data(
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-// main
-//
-// Main is the CLI entrypoint to the xml converter. It handles processing all
-// of the command line data into a format the internal functions want to
-// receive.
+// parse_arguments
+// Processes all of the command line data into a format the internal functions
+// want to receive.
 //
 // Example usage
-//   ./xml_converter --input-taco-paths ../packs/marker_pack --output-split-guildpoint-path ../output_packs
-//   ./xml_converter --input-taco-paths ../packs/* --output-split-guildpoint-path ../output_packs
+//   ./xml_converter --input-taco-paths ../packs/marker_pack --output-guildpoint-path ../output_packs
+//   ./xml_converter --input-taco-paths ../packs/* --output-guildpoint-path ../output_packs --split_by_map_id
+//
+// --import_FORMAT_paths <path> --FLAGS [--import_FORMAT_paths <path> --FLAGS]
+// --export_FORMAT_paths <path> --filter "conditions" --FLAGS
+// [--export_FORMAT_paths <path> --filter "conditions" --FLAGS]
```

Comment:
Are these examples?

Reply:
Its the template for the command line when we create new arguments. Would it make sense to label them correctly or do they not belong here?

Reply:
Do they belong here? I think they would be better served if they were visible to the user. Perhaps in a `--help` menu.

Reply:
I will move this to a help function in a future PR